themes/custom/droopler_theme/scss/components/_d_blog.scss

```diff
@@ -0,0 +1,262 @@
+$text-container-width: 35em !default;
+$img-width-for-desktop: 41em !default; //make img width the sames as text for desktop
+
+$font-color: $white !default;
+$headers-color: $black !default;
+$under-background-color: $black !default; // color when image is not
+$color-taxonomy-in-header: #8e8a89 !default;
```

Comment:
Please define a new color variable for this in _colors.scss - can be $color-aeneas